app/controllers/campaigns_controller.rb

```diff
@@ -0,0 +1,51 @@
+class CampaignsController < ApplicationController
+  def index
+    @campaigns = Campaign.all
+  end
+
+  def show
+    @campaign = Campaign.find(params[:id])
+  end
+  def edit
+    @campaign = Campaign.find(params[:id])
+  end
+
+  def update
+    @campaign = Campaign.find(params[:id])
+    @campaign.sent_at = DateTime.now
+
+    if @campaign.update(campaign_params)
+      redirect_to(@campaign)
+    else
+      render :edit, status: :unprocessable_entity
```

Comment:
You don't need to include status if his is not a json request. You are rendering a view and the status is implied